transcoder/message/MessageParser.py

```diff
@@ -210,6 +211,15 @@ def process_data(self):
         """Entry point for individual message processing"""
         with self.source:
             for raw_record in self.source.get_message_iterator():
+                # TODO
```

Comment:
Arguably more performant to do this, but doesn't flow as simply as keeping it in the DatacastParser.py